runtime/executor/src/main/java/org/apache/nemo/runtime/executor/data/stores/GlusterFileStore.java

```diff
@@ -83,7 +83,7 @@ public Block createBlock(final String blockId) {
    * @throws BlockWriteException if fail to write.
```

Comment:
Would you please remove the stale comment in L83?

Reply:
Sorry for the late response. It's done!